.travis.yml

```diff
@@ -1,16 +1,16 @@
+dist: latest
 language: python
 python:
-- 2.7
```

Comment:
We need to keep testing with Python 2.7 for a while (at least until edx-platform is successfully deployed in production with Python 3).

Reply:
Yes.